English.html

```diff
@@ -0,0 +1,223 @@
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://code.jquery.com/ui/1.13.0/themes/base/jquery-ui.css">
+  <script src="https://code.jquery.com/jquery-3.6.0.js"></script>
+  <script src="https://code.jquery.com/ui/1.13.0/jquery-ui.js"></script>
+  <script type="text/javascript" src="./words.js"></script>
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>  
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+<body>
+    <div class="container">
+        <h2>Spell Checker</h2>
+        <br/>
+        <ul class="nav nav-pills">
+          <li class="active"><a href="english.html">English</a></li>
+          <li><a href="irish.html"> Irish </a></li>
+        </ul>
+        <br/>
+        <br/>
+            <textarea class="form-control" id="tags"></textarea>
+            <br/>
+            <br/>
+            <br/>
+            <form>
+            <!-- <select class="form-select" multiple aria-label="multiple select example" id="list"></select> -->
+
+            <select class="form-select form-select-lg mb-3" aria-label=".form-select-lg example" size="20" id="blist"></select>
+
+          </form>
+      </div>
+      <script>
+          //<button onClick="submit()" class="btn btn-default">Verify</button>   
+    //     function submit(){
+    //     $.post("http://127.0.0.1:5000/en", {words:$("#tags").val()}, function(result){
+    //             $("#tags").val(result)
+    //   });
+    //     }
+
+        $(document).ready(function() {
+          $("#blist").hide();
+    console.log(words.length)
+    var availableTags = words
+    function split( val ) {
+      return val.split( / \s*/ );
+    }
+    function extractLast( term ) {
+        if(term.split(" ").pop()==""){
+            console.log(term)
+            return term
+         } 
+         return term.split(" ").pop();       
+    }
+ 
+    $("#tags")
+      .on( "change",
+       function( event ) {
+        if ( event.keyCode === $.ui.keyCode.TAB &&
+            $( this ).autocomplete( "instance" ).menu.active ) {
+          event.preventDefault();
+        }
+      }
+      )
+      .autocomplete({
+        minLength: 0,
```

Comment:
Is this user friendly?I mean that there is no description about how to operate the application?

Reply:
Now you can understand it easily.